distribution/test/itests/test-itests-common/src/test/java/ddf/common/test/mock/csw/FederatedCswMockServer.java

```diff
@@ -46,6 +46,8 @@
     private final String httpRoot;
 
     private final int port;
+    
```

Comment:
The `FederatedCswMockServer` doesn't need the metacard ID. We should be able to revert all those changes.